Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX_test.py

```diff
@@ -0,0 +1,423 @@
+
+GET_ANALYSIS_STATUS_CONTEXT = {
+    'csfalconx.resource(val.resource === obj.resource)':
+        {
+            'id': 'id',
+            'state': 'success',
+            'created_timestamp': '2020-03-16T17:04:48Z',
+            'environment_id': 160
+        }
+}
+
+CHECK_QUOTA_STATUS_HTTP_RESPONSE = {
+    'meta': {'query_time': 0.008237956, 'powered_by': 'falconx-api', 'trace_id': 'trace_id',
+             'quota': {'total': 100, 'used': 47, 'in_progress': 2}}, 'resources': None, 'errors': []}
+
+CHECK_QUOTA_STATUS_CONTEXT = {
+    'csfalconx.resource(val.resource === obj.resource)':
+        {
+            'total': 100,
+            'used': 47,
+            'in_progress': 2
+        }
+}
+
+FIND_SANDBOX_REPORTS_ARGS = {
+        "offset": "",
+        "limit": "",
+        "sort": "",
+        "filter": "",
+}
+
+FIND_SANDBOX_REPORTS_HTTP_RESPONSE = {
+    'meta': {'query_time': 0.008271345, 'pagination': {'offset': 0, 'limit': 10, 'total': 69},
+             'powered_by': 'falconx-api', 'trace_id': 'trace_id',
+             'quota': {'total': 100, 'used': 47, 'in_progress': 2}},
+    'resources': ['resources1',
+                  'resources2',
+                  'resources3',
+                  'resources4'], 'errors': []}
+
+FIND_SANDBOX_REPORTS_CONTEXT = {
+    'csfalconx.resource(val.resource === obj.resource)':
+        {
+            'resources': ['resources1', 'resources2', 'resources3', 'resources4']
+        }
+}
+
+FIND_SUBMISSION_ID_ARGS = {
+        "offset": "",
+        "limit": "",
+        "sort": "",
+        "filter": "",
+}
+
+FIND_SUBMISSION_ID_HTTP_RESPONSE = {'meta': {'query_time': 0.008812114, 'pagination': {'offset': 0, 'limit': 10, 'total': 72},
+                          'powered_by': 'falconx-api', 'trace_id': 'trace_id',
+                          'quota': {'total': 100, 'used': 47, 'in_progress': 2}},
+                            'resources': ['resources1',
+                                          'resources2',
+                                          'resources3',
+                                          'resources4'], 'errors': []}
+
+FIND_SUBMISSION_ID_CONTEXT = {
+    'csfalconx.resource(val.resource === obj.resource)':
+        {
+            'resources': ['resources1', 'resources2', 'resources3', 'resources4']
+        }
+}
+
+
+@pytest.mark.parametrize('command, args, http_response, context', [
+    (upload_file_command, UPLOAD_FILE_ARGS, UPLOAD_FILE_HTTP_RESPONSE, UPLOAD_FILE_CONTEXT),
+    (send_uploaded_file_to_sendbox_analysis_command, SEND_UPLOADED_FILE_TO_SENDBOX_ANALYSIS_ARGS, SEND_UPLOADED_FILE_TO_SENDBOX_ANALYSIS_HTTP_RESPONSE, SEND_UPLOADED_FILE_TO_SENDBOX_ANALYSIS_CONTEXT),
+    (send_url_to_sandbox_analysis_command, SEND_URL_TO_SANDBOX_ANALYSIS_ARGS, SEND_URL_TO_SANDBOX_ANALYSIS_HTTP_RESPONSE, SEND_URL_TO_SANDBOX_ANALYSIS_CONTEXT),
+    (get_full_report_command, GET_FULL_REPORT_ARGS, GET_FULL_REPORT_HTTP_RESPONSE, get_full_report_context),
```

Comment:
maybe we should have a few examples. a report for a file, report for url, report for a malicious file, etc.
this is to make sure that we map and parse all the data as needed.